File: Code-Katas/Python/SumArray.py
# Add every item from an array

def sum (numbers):

    count = [0]

    for i in range( len( numbers ) ):

        count.append( numbers[i] + count[i] )


    print( count[ len( count ) - 1 ] )
    return count[ len( count ) - 1 ] 

sum([]), '0'
sum([1, 5.2, 4, 0, -1]) , '9.2'

File: Code-Katas/Javascript/SumArray.js


function sum (numbers) {

    const count = [0]

    for ( var i = 0; i < numbers.length; i++ ) {
        if ( numbers == [] ) {
            return '0'
        } else {
            count.push( numbers[i] + count[ i ] )
        }
    }

    let results = count[ count.length - 1 ]
    console.log( results )
    return results
    
};

sum([]), '0'
sum([1, 5.2, 4, 0, -1]) , '9.2'